api/envoy/extensions/filters/http/oauth2/v3/oauth.proto

```diff
@@ -52,6 +52,10 @@ message OAuth2Credentials {
     // Cookie name to hold the refresh token. Defaults to ``RefreshToken``.
     string refresh_token = 5
         [(validate.rules).string = {well_known_regex: HTTP_HEADER_NAME ignore_empty: true}];
+
+    // Cookie name to hold the nonce value. Defaults to ``OauthNonce``.
+    string oauth_nonce = 6
+        [(validate.rules).string = {well_known_regex: HTTP_HEADER_NAME ignore_empty: true}];
```

Comment:
Enabling this by default enhances the security posture of the OAuth2 filter. This practice is also recommended by the OAuth2 spec: https://datatracker.ietf.org/doc/html/rfc6819#section-5.3.5.  So my preference is that we enable this since this release and don't provide an option to disable it unless there is a particular reason asking for it.

There are no side effects, as the nonce is transparent to both the end users and the applications.

Reply:
got it. Thanks.